### DISCUSSION TITLE
High availability option for the scheduler with an Azure SQL DB

### DISCUSSION THREAD

**Discussion by joeboctor** [apache/airflow#35114] [Q&A]
Hi, 

I am wondering if anyone has tried to run multiple schedulers with an Azure SQL DB.  The documentation is saying that it is supported but has been tested and supported on PostgreSQL and MySQL.  No tests have been done on Azure SQL.
If not, I am willing to do the testing for the community.  Is there a test suite that I run to ensure correct functionality?  I started the testing but I am not seeing any issues but I am wondering if there is comprehensive set of tests for the High availability scenarios.

Thanks

![image](https://github.com/apache/airflow/assets/108768564/3ac182a0-ba08-4c88-9b4e-59c224645dc8)

**Comment by Taragolis** [comment]
Support of Microsoft SQL Server as DB backend for Airflow will be discontinued soon:
- [Set up a Database Backend: MsSQL](https://airflow.apache.org/docs/apache-airflow/2.7.2/howto/set-up-database.html#setting-up-a-mssql-database)
- [Discussion on Dev-List](https://lists.apache.org/thread/r06j306hldg03g2my1pd4nyjxg78b3h4)
- [Voting Results](https://lists.apache.org/thread/pgcgmhf6560k8jbsmz8nlyoxosvltph2) - Drop: 21 votes (14 binding, 7 non-binding) vs Keep: 0 votes
